icu4c/source/i18n/measunit_impl.h

```diff
@@ -189,6 +196,16 @@ struct U_I18N_API MeasureUnitImpl : public UMemory {
      */
     MeasureUnitImpl copy(UErrorCode& status) const;
```

Comment:
I noticed this PR (with the change originating in https://github.com/icu-units/icu/pull/15) is adding a copy constructor, in conflict with this copy() method's intent to keep copying explicit: "Don't use copy constructor to make this explicit." I filed https://github.com/icu-units/icu/issues/113